app/javascript/app/pages/national-circumstances/economy/gdp/gdp-component.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import ModalMetadata from 'components/modal-metadata';
+import { TabletLandscape, TabletPortraitOnly } from 'components/responsive';
+import InfoDownloadToolbox from 'components/info-download-toolbox';
+import { Dropdown, Chart } from 'cw-components';
+import MetaProvider from 'providers/metadata-provider';
+import GdpProvider from 'providers/gdp-provider';
+import WorldBankProvider from 'providers/world-bank-provider';
+import { CustomYAxisTick } from './axis-ticks';
+
+import styles from './gdp-styles';
+
+class GDP extends PureComponent {
+  handleInfoClick = () => {
+    this.props.setModalMetadata({
+      slugs: 'historical_emissions_cait',
+      open: true
+    });
+  };
+
+  handleMetricChange = metric => {
+    this.props.updateMetricSelected({
+      section: 'economy',
+      query: { metric: metric.value }
+    });
+  };
+
+  handleDownloadClick = () => {
+    console.info('TODO: link todownload data endpoint', this.props);
+  };
+
+  render() {
+    const { metricSelected, metricOptions, gdpParams, chartData } = this.props;
+
+    const dropdown = (
+      <Dropdown
+        theme={{ wrapper: styles.dropdown }}
+        options={metricOptions}
+        value={metricSelected}
+        onValueChange={this.handleMetricChange}
+        hideResetButton
+      />
+    );
+    const toolbar = (
+      <InfoDownloadToolbox
```

Comment:
Can we disable the download button until it has its functionality ready?

Reply:
sure